doc/index.md

```diff
@@ -1,10 +1,55 @@
-[![travis](https://img.shields.io/travis/npgsql/npgsql.svg?label=travis)](https://travis-ci.org/npgsql/Npgsql.EntityFrameworkCore.PostgreSQL)
 [![gitter](https://img.shields.io/badge/gitter-join%20chat-brightgreen.svg)](https://gitter.im/npgsql/npgsql)
 
+# Build Status
+
+| Branch                               | Build Status                                     | Stable                                 | Next Patch                                  | Unstable                                                    |
+|:-------------------------------------|:-------------------------------------------------|:---------------------------------------|:--------------------------------------------|:------------------------------------------------------------|
+| [`dev`][tree-dev]                    | [![status][build-dev]][ci-dev]                   | [![stable][nuget-vpre]][nuget-npgsql]  |                                             | [![unstable][npgsql-unstable-vpre]][myget-npgsql-unstable]  |
+| [`master`][tree-master]              | [![status][build-master]][ci-master]             | [![stable][nuget-v]][nuget-npgsql]     | [![next patch][npgsql-vpre]][myget-npgsql]  |                                                             |
+| [`hotfix/2.2.6`][tree-hotfix/2.2.*]  | [![status][build-hotfix/2.2.*]][ci-hotfix/2.2.*] | [![stable][nuget-2.2.*]][nuget-npgsql] | [![next patch][npgsql-2.2.*]][myget-npgsql] | [![unstable][npgsql-unstable-2.2.*]][myget-npgsql-unstable] |
+| [`hotfix/2.1.11`][tree-hotfix/2.1.*] | [![status][build-hotfix/2.1.*]][ci-hotfix/2.1.*] | [![stable][nuget-2.1.*]][nuget-npgsql] | [![next patch][npgsql-2.1.*]][myget-npgsql] | [![unstable][npgsql-unstable-2.1.*]][myget-npgsql-unstable] |
+
+
+[* Package Feeds]: <>
+[nuget-npgsql]:          https://www.nuget.org/packages/Npgsql.EntityFrameworkCore.PostgreSQL
+[myget-npgsql]:          https://www.myget.org/feed/npgsql/package/nuget/Npgsql.EntityFrameworkCore.PostgreSQL
+[myget-npgsql-unstable]: https://www.myget.org/feed/npgsql-unstable/package/nuget/Npgsql.EntityFrameworkCore.PostgreSQL
+
+[* Branches]: <>
+[tree-dev]: https://github.com/npgsql/Npgsql.EntityFrameworkCore.PostgreSQL/tree/dev
+[tree-master]: https://github.com/npgsql/Npgsql.EntityFrameworkCore.PostgreSQL/tree/master
+[tree-hotfix/2.2.*]: https://github.com/npgsql/Npgsql.EntityFrameworkCore.PostgreSQL/tree/hotfix/2.2.6
+[tree-hotfix/2.1.*]: https://github.com/npgsql/Npgsql.EntityFrameworkCore.PostgreSQL/tree/hotfix/2.1.11
+
+[* Azure Pipelines (badges)]: <>
+[build-dev]:          https://dev.azure.com/npgsql/EFCore.PG/_apis/build/status/npgsql.Npgsql.EntityFrameworkCore.PostgreSQL?branchName=dev
+[build-master]:       https://dev.azure.com/npgsql/EFCore.PG/_apis/build/status/npgsql.Npgsql.EntityFrameworkCore.PostgreSQL?branchName=master
+[build-hotfix/2.2.*]: https://dev.azure.com/npgsql/EFCore.PG/_apis/build/status/npgsql.Npgsql.EntityFrameworkCore.PostgreSQL?branchName=hotfix/2.2.6
```

Comment:
Once again, I really think anything with an explicit version in it (especially patch versions) is unfortunately never going to be up to date...